src/components/new-request/name-check/name-check.vue

```diff
@@ -0,0 +1,809 @@
+<template>
+  <v-container id="name-check-main-container" class="white pa-0 rounded">
+    <name-check-issues-dialog :attach="'#app'"
```

Comment:
I think you can just use `attach="#app"` here.

Reply:
ha, yes